crates/ruff_python_parser/src/string_token_flags.rs

```diff
@@ -201,33 +232,45 @@ impl StringKind {
         }
     }
 
-    /// A `str` representation of the prefixes used (if any)
-    /// in the string's opener.
-    pub const fn prefix_str(self) -> &'static str {
+    pub const fn prefix(self) -> StringPrefix {
```

Comment:
Nit: It could make sense to move this to `StringPrefix:from_kind` so that the method is next to `StringPrefix::as_flags`

Reply:
Nice idea!